src/deepak/MaxRepeatingWordWithFreq.java

```diff
@@ -0,0 +1,36 @@
+/* max repeating word with its freq.
+input : Hi Hello Hi Techno Hello Hi
+output : Hi -> 3 */
+
+package deepak;
+
+public class MaxRepeatingWordWithFreq {
+	
+	static void freq(String str) {
+		String [] input=str.split(" ");
+		int max = 0;
+		String maxName = null; 
+		for (int index = 0; index < input.length;index++) {
+			int count = 0;
+			String word = input[index];	
+			if (str.contains(word)) {
```

Comment:
Here you are putting one word from array to the variable called "word", & then you are checking that variable value into array, so here think logically, it will always gives you true right?
storing in one variable and checking same variable in that array makes no sense here.